src/Game/Models/Game.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Game\Models;
+
+use Game\AbstractGameClass;
+use Game\BoardInterface;
+use Game\PlayerInterface;
+
+/**
+ * Class Game
+ * @package Game\Model
+ */
+class Game extends AbstractGameClass
```

Comment:
No need to add suffix Class - everybody knows it is really class. Please rename to AbstractGame

Reply:
+